common/client.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// Client holds configuration data associated with the HTTP(s) session
+type Client struct {
+	HTTPClient http.Client
+	// TODO(tho) API tokens and client credentials
+}
+
+// NewClient instantiates a new Client
+func NewClient() *Client {
+	return &Client{
+		HTTPClient: http.Client{
+			Timeout: 5 * time.Second,
+		},
+	}
+}
+
```

Comment:
Would be good to have the comments on each of the Public Methods atleast!

Reply:
The methods in the `common` package are only public because they are shared by both provisioning and verification APIs.  They are not meant to be used by the general public.  For that kind of users, the comments on the public methods and data in the `verification` and `provisioning` packages, as well as the two `docs.go` files apply.